locale/en/corteza-webapp-compose/block.yaml

```diff
@@ -369,6 +369,18 @@ recordList:
     conditions:
       and: AND
       or: OR
+      nil: "NULL"
```

Comment:
you're using `.null` somewhere so you need to change it to that here too

NULL is not really a condition, its more of a value. 
Doesnt matter where you move it honestly, just not under conditions.